src/bundle/Resources/public/js/CKEditor/core/base-ckeditor.js

```diff
@@ -99,8 +108,53 @@ import Italic from '../../../../../../../../ezplatform-admin-ui-assets/Resources
 
             InlineEditor.create(container, {
                 initialData: section.innerHTML,
-                plugins: [CharacterCounter, Essentials, Paragraph, Bold, Italic],
-                toolbar: ['bold', 'italic'],
+                plugins: [
+                    CharacterCounter,
+                    Essentials,
+                    Heading,
+                    Alignment,
+                    ListStyle,
+                    Table,
+                    TableToolbar,
+                    Bold,
+                    Italic,
+                    Underline,
+                    Subscript,
+                    Superscript,
+                    Strikethrough,
+                    BlockQuote,
+                ],
+                toolbar: [
+                    'heading',
+                    '|',
+                    'alignment',
+                    '|',
+                    'bulletedList',
+                    'numberedList',
+                    'insertTable',
+                    '|',
+                    'bold',
+                    'italic',
+                    'underline',
+                    'subscript',
+                    'superscript',
+                    'strikethrough',
+                    'blockQuote',
+                ],
+                heading: {
+                    options: [
+                        { model: 'paragraph', title: 'Paragraph', class: 'ck-heading_paragraph' },
```

Comment:
no translations?

Reply:
These config options will not be here for sure currently don't know where the translation will come from so I would keep it as it is for the time being.